services/core-api/app/api/mines/explosives_permit/models/explosives_permit_document_type.py

```diff
@@ -48,6 +52,27 @@ def validate_issuing_inspector(explosives_permit):
                 raise Exception('No signature for the Issuing Inspector has been provided')
 
         def create_image(source, width=None, height=None):
+
+            # If there is a prefix in the source, remove it
+            base64_source = source
```

Comment:
nice

Reply:
Yeah, took a bit of figuring to get it working just right.